python/cudf/cudf/core/series.py

```diff
@@ -3114,6 +3115,13 @@ def describe(
     ):
         """{docstring}"""
 
+        if datetime_is_numeric:
+            warnings.warn(
```

Comment:
I'm confused: if we are inside the `if`, does it not mean that `datetime_is_numeric` is already `True`?

Reply:
My bad, fixed it.